ext/mysql2/result.h

```diff
@@ -1,5 +1,6 @@
 #ifndef MYSQL2_RESULT_H
 #define MYSQL2_RESULT_H
+#include <stdbool.h>
```

Comment:
Sorry I did not find the information if Ruby requires C extensions compiled with C89.